$NetBSD$

Disable stuff that is not supported by the latest autoconf.

--- makefiles/configure.in~	2005-01-30 10:28:00.000000000 +0000
+++ makefiles/configure.in
@@ -15,10 +15,10 @@ AC_SUBST(VERSION)
 dnl Checks for programs.
 AC_PROG_CC
 AC_ISC_POSIX
-AM_C_PROTOTYPES
-if test "x$U" != "x"; then
-  AC_MSG_ERROR(Compiler not ANSI compliant)
-fi
+dnl AM_C_PROTOTYPES
+dnl if test "x$U" != "x"; then
+dnl   AC_MSG_ERROR(Compiler not ANSI compliant)
+dnl fi
 AM_PROG_LIBTOOL
 AC_PROG_INSTALL
 
